### DISCUSSION TITLE
Is hpool pool supported

### DISCUSSION THREAD

**Discussion by lifejx** [madMAx43v3r/chia-plotter#423] [General]
When I use the poolkey of HPOOL and the corresponding farmerkey, invalid < poolkey >: and invalid < farmerkey >: (needs to be 48 bytes) appear. In win10 platform.How to solve it

**Comment by DrBranom** [comment]
hi. Go to folder C:\Users\*User*\.chia\mainnet\plotter
in the plotter_log*.txt file you will see the correct pool public key and farmer public key.

**Reply by lifejx** [reply]
thank you! The problem has been solved
